metrics/metrics-tests/src/test/java/org/apache/james/metrics/tests/RecordingMetricTest.java

```diff
@@ -36,4 +40,24 @@ void setUp() {
     public Metric testee() {
         return testee;
     }
+
+    @Disabled("This metric doesn't allow negative counter value")
+    @Test
+    public void getCountShouldReturnZeroWhenCounterIsNegative() {
```

Comment:
This should not be part of the contract (but in each test) as we have some diverging behaviour.